pkg/cmd/cli/cmd/version.go

```diff
@@ -126,13 +126,14 @@ func (o VersionOptions) RunVersion() error {
 		}
 
 		ocVersionBody, err := oClient.Get().AbsPath("/version/openshift").Do().Raw()
-		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err)) {
+		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err) || kapierrors.IsForbidden(err)) {
```

Comment:
in the case of a not found / unauthorized / forbidden error, we should not continue on

if kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err) || kapierrors.IsForbidden(err) {
  // handle unknown server version
  return
}
if err != nil {
  done <- err
  return
}
... parse json, output error or parsed version ...
```